test/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   env: {
-    mocha: true,
-    node: true,
```

Comment:
Why are these changes needed?

Reply:
Left-overs from copying the test directory. Fixed pushed in e895117